ydb/docs/en/core/security/audit-log.md

```diff
@@ -1,56 +1,314 @@
 # Audit log
 
-_An audit log_ is a stream that includes data about all the operations that tried to change the {{ ydb-short-name }} objects, successfully or unsuccessfully:
+_An audit log_ is a stream of records that document the operation of the {{ ydb-short-name }} cluster. Unlike technical logs, which help detect failures and troubleshoot issues, the audit log provides data relevant to security. It serves as a source of information that answers the questions: who did what, when, and from where.
```

Comment:
The new first sentence is too generic. A much broader scope can be considered "cluster operation" than what's explained further.